twine/cli.py

```diff
@@ -12,29 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import argparse
-from typing import Any, Dict, List, Tuple
+from typing import Any, List, Tuple
 
 import pkginfo
 import requests
 import requests_toolbelt
 import setuptools
 import tqdm
+from importlib_metadata import entry_points
 
 import twine
 from twine import _installed
-from twine import importlib_metadata
```

Comment:
Can we ditch https://github.com/pypa/twine/blob/c1807fac1275796cc8cbfbc29ce55596cd32ac80/twine/__init__.py#L29-L34 altogether? I understand the `metadata` API hasn't changed and we could update teh version check there eventually for 3.10 and then do this dance but:

1. I don't like putting compatibility shims like that into `twine.__init__` and then importing them here
2. If we're unconditionally requiring the backport, we may as well not confuse people with that code.

2 is important to me because I know at the very least the Gentoo redistributors will come for us with pitchforks for using a backport even though it's absolutely justified here and I don't want that code there to confuse them or others.

Reply:
That was my instinct too, but I was limiting the scope of my change. Changed as recommended in 92c2fc4.